includes/services/GoogleMaps3/Maps_GoogleMaps3.php

```diff
@@ -222,60 +223,60 @@ public function addParameterInfo( array &$params ) {
 			'message' => 'maps-googlemaps3-par-enable-fullscreen',
 		);
 	}
-	
+
 	/**
 	 * @see iMappingService::getDefaultZoom
-	 * 
+	 *
 	 * @since 0.6.5
-	 */	
+	 */
 	public function getDefaultZoom() {
 		global $egMapsGMaps3Zoom;
 		return $egMapsGMaps3Zoom;
-	}	
-	
+	}
+
 	/**
 	 * @see MapsMappingService::getMapId
-	 * 
+	 *
 	 * @since 0.6.5
 	 */
 	public function getMapId( $increment = true ) {
 		static $mapsOnThisPage = 0;
-		
+
 		if ( $increment ) {
 			$mapsOnThisPage++;
 		}
-		
+
 		return 'map_google3_' . $mapsOnThisPage;
-	}	
-	
+	}
+
 	/**
 	 * Returns the names of all supported map types.
-	 * 
+	 *
 	 * @return array
 	 */
 	public static function getTypeNames() {
 		return array_keys( self::$mapTypes );
 	}
-	
+
 	/**
 	 * Returns the names of all supported map layers.
-	 * 
+	 *
 	 * @since 1.0
-	 * 
+	 *
 	 * @return array
 	 */
 	public static function getLayerNames() {
 		return self::$mapLayers;
-	}	
-	
+	}
+
 	/**
 	 * @see MapsMappingService::getDependencies
-	 * 
+	 *
 	 * @return array
 	 */
 	protected function getDependencies() {
 		return array(
-			self::getApiScript( $GLOBALS['wgLang']->getCode() ),
+			self::getApiScript( $GLOBALS['wgContLang']->getCode() ),
 		);
```

Comment:
To ensure you don't run into any issues with before/after LocalSettings being available add

`Maps_Settings.php`

```
+		$GLOBALS['egMapsGMaps3Language'] = '';
```
and extend `Maps.php` with

```
 	$GLOBALS['wgExtensionFunctions'][] = function () {
 		Hooks::run( 'MappingServiceLoad' );
 		Hooks::run( 'MappingFeatureLoad' );
 
+		if ( $GLOBALS['egMapsGMaps3Language'] === '' ) {
+			$GLOBALS['egMapsGMaps3Language'] = $GLOBALS['wgLang'];
+		}
+
```